spec/unit/provider/sensu_entity/sensuctl_spec.rb

```diff
@@ -8,12 +8,12 @@
 
   describe 'self.instances' do
     it 'should create instances' do
-      allow(@provider).to receive(:sensuctl_list).with('entity').and_return(my_fixture_read('entity_list.json'))
+      allow(@provider).to receive(:sensuctl_list).with('entity').and_return(JSON.parse(my_fixture_read('entity_list.json')))
```

Comment:
why did we just read the json before and now we read and then parse?

Reply:
Because before `sensuctl_list` would return a JSON string that had to be parsed in each provider.  That parsing of the string was moved to shared function so now that function returns parsed JSON so rather than rewrite how the tests are written I just parse the same fixtures.